src/controller/cap-level-controller.js

```diff
@@ -127,7 +127,7 @@ class CapLevelController extends EventHandler {
     let width;
     const media = this.media;
     if (media) {
-      width = media.width || media.clientWidth || media.offsetWidth;
+      width = Math.max(media.width, media.clientWidth, media.offsetWidth);
```

Comment:
```suggestion
      width = Math.max(media.width, media.clientWidth || media.offsetWidth);
```

> I have force-pushed the PR branch to a slightly different solution to use the maximum of all the dimension values after noticing that the previous solution broke firefox+linux.

Is this intentional? You didn't do the same for height.

Reply:
Actually it was not intentional, I meant to have the same but missed to replace one `||` with `,`
In my mind, there is no net difference, but just using `Math.max()` is simpler, so I would suggest to only use that.
Change pushed